src/test/java/com/wikia/webdriver/common/templates/NewTestTemplate.java

```diff
@@ -15,6 +16,7 @@
 import org.testng.annotations.BeforeMethod;
 
 import java.lang.reflect.Method;
+import java.sql.Driver;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.sql.Driver'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)